test/e2e/performanceprofile/functests/utils/consts.go

```diff
@@ -24,7 +26,16 @@ var NodesSelector string
 // ProfileNotFound is true when discovery mode is enabled and no valid profile was found
 var ProfileNotFound bool
 
+// Complete file name for the latency tools executable
+var LatencyToolsPath string
+
 func init() {
+
+	LatencyToolsPath = os.Getenv("LATENCY_TOOLS_PATH")
```

Comment:
sounds a good idea indeed. On the other hand, if the path did change, and test executable was compiled, it will be found only in the default path even though the env var LATENCY_TOOLS_PATH is set.  
A case that I can think of where this can be easily broken is running 5_latency_testing after specifying a new non-default path under LATENCY_TOOLS_PATH. this will result to skip the tests because the test executable was not found in the new path, but in fact in the default one because this https://github.com/openshift/cluster-node-tuning-operator/blob/master/hack/build-latency-test-bin.sh#L10 is kept as is. so test executable is generated but tests are skipped because they search for it in the wrong directory. Did you have a chance to test this?

Reply:
Yes, I totally agree. 
If you run the test suite with a wrong environment it would fail, but I think that is applicable to any other env var.

Error logging can be improved so the problem, in case of happening, will be easier to debug.
I could export the `LATENCY_TOOLS_PATH` in `build-latency-test.sh` after the build but I am not sure if that would help at all.

One thing is certain ... if `LATENCY_TOOLS_PATH` is not set or empty everything should work as expected.